src/components/BooksList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
- [ ] it is placed in src/containers directory. 

We need to use presentational and container components pattern which separates components into the ones that connects to the Redux store (container components) and the ones that doesn't(presentational components). Check out [this](https://react-redux.js.org/5.x/introduction/why-use-react-redux#it-encourages-good-react-architecture) from redux docs.

Check this [article](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) for more info.

Reply:
Thanks for your suggestion. It is implemented.